app/assets/stylesheets/foundation_and_overrides.scss

```
@@ -132,7 +132,7 @@ $google-color: #e22522;
 $facebook-color: #45619d;
 
 
-// $alert-color: #f04124;
+$alert-color: #AC230C;
```

I find following the AAA level rule too strict in this case :thinking:, since psycologically we lose a bit of the effect the red color produces. So how about:

```suggestion
$alert-color: #D22C0F;
```

---

Done!